backstory/src/main/java/com/google/sps/story/data/DatamuseRequestClient.java

```diff
@@ -37,6 +37,10 @@
   /** holds the base url to query */
   private final String url;
 
+  /** a list of topics related to storytelling for which to filter adjectives/gerunds for */
+  public static final String[] STORYTELLING_TOPICS = {"story", "fairytale", "narrative", "anecdote",
+      "drama", "fantasy", "adventure", "poem", "grand"};
```

Comment:
Consider using an immutable List instead to avoid the possibility of the values in the array changing:
http://www.technology.heartland.edu/courses/Computer%20Science/Programming/Java%20Courses/JTest%20User's%20Guide/mispsfa.htm

Reply:
That makes sense. I'll make the change.

Reply:
Done